lib/core-integration/src/Test/Integration/Framework/DSL.hs

```diff
@@ -832,6 +835,13 @@ emptyWalletWith ctx (name, passphrase, addrPoolGap) = do
     expectResponseCode @IO HTTP.status202 r
     return (getFromResponse id r)
 
+
+-- | Restore a faucet and wait until funds are available.
+fixtureByronWallet
+    :: Context t
+    -> IO ApiByronWallet
+fixtureByronWallet _ = undefined
```

Comment:
something more meaningful like `error "fixtureByronWallet: not yet implemented"` would be better.

Reply:
corrected